.github/workflows/spark.yaml

```diff
@@ -6,10 +6,12 @@ on:
     branches:
       - master
```

Comment:
I assume this workflow is for building and publishing, right?
Suggestion: rename workflow name to spark-client